erts/etc/common/inet_gethost.c

```diff
@@ -18,10 +18,10 @@
  * %CopyrightEnd%
  */
 /*
- * Erlang port program to do the name service lookup for the erlang 
+ * Erlang port program to do the name service lookup for the erlang
  * distribution and inet part of the kernel.
  * A pool of subprocess is kept, to which a pair of pipes is connected.
- * The main process schedules requests among the different subprocesses 
```

Comment:
We appreciate spelling typos, but we don't fix white space errors in otherwise untouched lines.

Please remove those changes. They seem to mainly be in this file erts/etc/common/inet_gethost.c.